src/applications/ivc-champva/10-7959a/tests/unit/containers/ConfirmationPage.unit.spec.js

```diff
@@ -24,7 +24,7 @@ const storeBase = {
   },
 };
 
-const storeBaseInvalid = {
+const storeBaseTruncated = {
```

Comment:
Confusingly named, as it was valid data. Renamed to indicate that it's a shorter version of the test data used above.